### PR TITLE
add script to pull contribution data for each group

### PR DESCRIPTION
**Pull request by boneskull**
- add migration for `data` col
- update `Group` model

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
     "node-slack": "0.0.7",
     "node-uuid": "1.4.7",
     "nodemailer": "1.11.0",
+    "opencollective-jobs": "github:opencollective/opencollective-jobs#be6d0a2014e36cdd073a5dda2b1f81d71ce07a7d",
```
Comment (asood123) [inline package.json]: Should we remove the code at the end?

Comment (boneskull) [inline package.json]: that references a particular commit.  if we remove it, we'll get whatever the `master` branch happens to be when `npm install` is run.  so, no, we  want to keep it until we start doing releases for that project

Comment (asood123) [inline package.json]: What about using version numbers instead of commits? Easier to identify and keep track of.

Comment (boneskull) [inline package.json]: I haven't tagged anything yet, but I can do that.

Comment (boneskull) [inline package.json]: done

File: scripts/update-contributions.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      })
+      .then(data => {
+        group.settings = group.settings || {};
+        group.settings.githubOrg = org;
+        group.data = data;
+        return group.save();
+      })
+      .then(() => {
+        console.log(`Successfully updated contribution data for group "${group.name}"`);
+      })
```
Comment (sedubois) [inline scripts/update-contributions.js]: this could be a one-liner and return value not used so can use tap `.tap(() => console.log(...))`

Comment (asood123) [inline scripts/update-contributions.js]: Btw, we should probably call the field inside `data` as `githubContributors`, so we can add other stuff to `data` later.

**Pull request merged**